backend/src/BuiltinExecution/Libs/Packages.fs

```diff
@@ -41,8 +41,8 @@ let parseGenericName (name : string) : GenericName =
     match List.rev modulesAndName with
     | name :: modulesInReverse ->
       { owner = owner; modules = List.rev modulesInReverse; name = name }
-    | [] -> failwithf "Invalid name (no name): %s" name
-  | [] -> failwithf "Invalid name (no owner): %s" name
+    | [] -> Exception.raiseInternal $"Invalid name (no name): {name}" []
+  | [] -> Exception.raiseInternal $"Invalid name (no owner): {name}" []
```

Comment:
When using `.raiseInternal` it's better to include pieces of data in the metadata [] rather than use string interpolation, for consistency (and also because then you can, when looking at all logs, more easily collect all logs of the same type.